lib/components/Nav.jsx

```diff
@@ -18,6 +18,26 @@ export const Nav = (props) => {
     <nav
       className='justify-end items-center hidden sm:flex w-2/3'
     >
+      <Link
+        href='https://www.notion.so/PoolTogether-Knowledge-Base-fa721ccefa3242eaabd125a8415acd27'
+        as='https://www.notion.so/PoolTogether-Knowledge-Base-fa721ccefa3242eaabd125a8415acd27'
+        shallow
+      >
+        <a
```

Comment:
I believe this would be better suited for the <Footer />. Personally I try to leave the top nav items as internal links so the navigation is always consistent (ie. you click "Developers" and the nav at the top stays the same. The footer is a better place for external links to other apps/sites. And/or in the content of the pages itself. I believe @Lililashka is working on a redesign of this site right now, though! Perhaps it would be best to include her in discussions around having links to the new Knowledge Base integrated into the landing site.

Reply:
Hey @chuckbergeron thanks for the feedback.
Still not very practical with React.
We can have a chat with @Lililashka about the placement but we could put it in the footer for now.